<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>420</height>
   </rect>
  </property>
  <property name="font">
   <font/>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
background-color: rgba(255, 204, 204, 1)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pinkButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>191</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>üòù</string>
    </property>
   </widget>
   <widget class="QPushButton" name="purpleButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>201</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>ü¶ß</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#startButton{
background-color: rgba(204, 255, 204, 1)
}</string>
    </property>
    <property name="text">
     <string>Normie modeüßç</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="sickoModeButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#sickoModeButton{
background-color: rgba(255, 204, 51, 1)
}</string>
    </property>
    <property name="text">
     <string>Sicko ModeüíÄ</string>
    </property>
   </widget>
   <widget class="QLabel" name="youLoseLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>You Lose !! üòà</string>
    </property>
   </widget>
   <widget class="QLabel" name="tryAgainLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>330</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>select game mode to keep playing</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>476</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
